benches/README.md

```diff
@@ -0,0 +1,6 @@
+# Getting started with benchmarking
+
+To run the benchmarks you need a nightly rust toolchain.
+Then you launch it with
+
+    rustup run nightly cargo bench --features=benchmarks
```

Comment:
Couldn't you just do `cargo +nightly bench --features/benchmarks`?

Reply:
Wasn't aware of this syntax. I'll update PR

Reply:
Sorted